charts/govuk-rails-app/templates/worker-deployment.yaml

```diff
@@ -26,6 +26,10 @@ spec:
       containers:
         - name: app
           image: "{{ .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
+          imagePullPolicy: {{ .Values.appImage.pullPolicy | default "Always" }}
```

Comment:
I was dubious about specifying a default here, but then found it's consistent with what we already have in `deployment.yaml` so 🤷👍

Reply:
yeah, I copied it across. Thanks!